XamlControlsGallery/ControlPages/StandardUICommandPage.xaml.cs

```diff
@@ -96,7 +96,12 @@ private void ListViewRight_ContainerContentChanging(ListViewBase sender, Contain
         {
             MenuFlyout flyout = new MenuFlyout();
             ListItemData data = (ListItemData)args.Item;
-            MenuFlyoutItem item = new MenuFlyoutItem() { Command = data.Command};
+            MenuFlyoutItem item = new MenuFlyoutItem() { Command = data.Command };
+            flyout.Opened += delegate (object element, object e) {
+                var flyOutElement = element as MenuFlyout;
+                var elementToHighlight = flyOutElement.Target as ListViewItem;
```

Comment:
```suggestion
                ListViewItem elementToHighlight = flyoutElement.Target as ListViewItem;
```

Similar feedback - declaring the type instead of using var feels more clear.

Reply:
Again: I agree. Thank you for the feedback! ^^